### ISSUE TITLE
Searching for verses with Strongs’ numbers

### ISSUE THREAD

**Issue by Noitarud** [AndBible/and-bible#2582]
I would like to search for numbers, if its implemented show me how do I locate verses with "5315" and "blood"?

**Comment by bvahedy** [comment]
In the find window type: 
strong:H5315

**Comment by tuomas2** [comment]
I believe this is covered on some ticket already but some sort of UI for this should be done.

**Comment by bvahedy** [comment]
I agree. You could add a check box for Strong's to the existing selection on find window and if it's checked, it adds the "strong:" behind the scene. Then if the user selects OT on the radio buttons programmatically you can prefix the number with "H" and  "G" for NT. This way the user just enters the number in the search window.

**Comment by agrogers** [comment]
I implemented this 18 months ago. You can see my attempt here:
https://github.com/AndBible/and-bible/pull/2019

Although I think it is good to have, I rarely use it.
___________________________
*www.TenOutOfTen.org* <https://www.TenOutOfTen.org>
***@***.***
(+61) 0477 174 193 (Australia)
(+95) 09 250018669 (Myanmar)

**Comment by bvahedy** [comment]
It would be good If we could implement at least the Strong's search portions of your mods into the alpha build.

**Comment by Noitarud** [comment]
Excellent.

Could have a button "strongs reference" and that could then paste in "strong:" and a pop up message to tell to preface a H(ebrew) or G(reek). Since that is probably not the only one (link please of others), have a "tools" drop-down with it and the others.

**Comment by Noitarud** [comment]
I will note one minor bug if its going to be left as is:
"strong:1234" only works if the user interface does not auto-capitalise the first letter of the search box.

**Comment by Noitarud** [comment]
My last post disappeared? It said if I selected a verse in a (strongs) search, then from another window made another strongs search then back out/select there, then pressed back on the first mentioned window I would see the objects of the second search highlighted (if they happen to be there) instead - can be useful but thought to note it.

The reason for today's post is it will not highlight if number H3068 was the search object.
